full-stack-tests/backend/src/integration/_setup.test.ts

```diff
@@ -33,7 +33,7 @@ before(async () => {
   const cfg = new IModelHostConfiguration();
   cfg.cacheDir = path.join(__dirname, ".cache");  // Set the cache dir to be under the lib directory.
   const iModelClient = new IModelsClient({ api: { baseUrl: `https://${process.env.IMJS_URL_PREFIX ?? ""}api.bentley.com/imodels`}});
-  cfg.hubAccess = new BackendIModelsAccess(iModelClient);
+  cfg.hubAccess = new BackendIModelsAccess(iModelClient) as any;
```

Comment:
Next time we should probably have a temporary @internal adapter class for such changes, I imagined TS would not complain :)

Reply:
Question is where would we store such class ? 

I assume it would be a wrapper for `BackendIModelsAccess` and as a result it would depend on _@itwin/imodels-access-backend_. Since test-apps and full-stack-tests depend on _@itwin/imodels-access-backend_ separately, such a class would be duplicated everywhere.

Reply:
We could keep it as @internal in core-backend.